packages/block-library/src/post-comment-author/index.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comment-author` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comment-author` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Return the post comment's content.
+ */
+function render_block_core_post_comment_author( $attributes, $content, $block ) {
+	$commentId = null;
+
+	if ( ! empty( $block->context['commentId'] ) ) {
+		$commentId = $block->context['commentId'];
+	}
+
+	if ( ! empty( $attributes['commentId'] ) ) {
+		$commentId = $attributes['commentId'];
+	}
```

Comment:
Do we need this? We didn't in `post-comment-content`, did we? I'm a bit concerned it might be confusing when this is used vs when `$block->context['commentId']` is used (does it have to do with the `ServerSideRender` component?)

Reply:
Yes, that's only needed for the ServerSideRenderer. Got rid of these after moving to client side implementation